src/main/java/bowlingexercise3/Bowling.java

```diff
@@ -57,6 +60,11 @@ public int score() {
         return score;
     }
 
+    //Private Boolean created to keep track of pointers which correctly add scores
+    private boolean Strike(int cursor) {
```

Comment:
A small thing, but the convention is for methods to start with a lowercase character.

Reply:
Addressed